src/ucp/proto/proto_am.inl

```diff
@@ -28,12 +28,19 @@ ucp_do_am_bcopy_single(uct_pending_req_t *self, uint8_t am_id,
 {
     ucp_request_t *req = ucs_container_of(self, ucp_request_t, send.uct);
     ucp_ep_t *ep       = req->send.ep;
+    size_t offset      = req->send.state.dt.offset;
     ssize_t packed_len;
 
     req->send.lane = ucp_ep_get_am_lane(ep);
     packed_len     = uct_ep_am_bcopy(ep->uct_eps[req->send.lane], am_id, pack_cb,
                                      req, 0);
-    if (packed_len < 0) {
+    ucs_assertv((packed_len < 0) ||
```

Comment:
first check is not needed, the second covers it

Reply:
it seems max_bcopy==INF (-1) - will break it